javascript-modal/styles.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.container {
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  visibility: hidden;
```

Comment:
`visibility` is generally not the preferred way to make elements hidden, since they still have size in the document flow while invisible. Rather than having all of your elements that could potentially be hidden be invisible by default, you should have one class that s responsible for making elements hidden using `display: none;`, and then just apply that class to elements when you need them to be invisible. If you want them to be invisible when the page loads, you can simply add that class to the element in your HTML document :+1: